internal/expr/expr_test.go

```diff
@@ -543,27 +543,27 @@ func (s *ExprSuite) TestPrepareMapError(c *C) {
 		"all output into map star",
 		"SELECT &M.* FROM person WHERE name = 'Fred'",
 		[]any{sqlair.M{}},
-		"cannot prepare expression: &M.* cannot be used for maps when no column names are specified",
+		"cannot prepare statement: columns must be specified for asterisk map in output expression: &M.*",
```

Comment:
'Asterisk map' sounds a little strange of me (it sounds like a strange data structure containing only asterisks). When I read `&M.*`, I hear 'm-dot-star.' Perhaps this error could read 'map-star' or 'starred map'?

Reply:
Changed.